packages/eas-cli/src/metadata/apple/tasks/app-info.ts

```diff
@@ -0,0 +1,88 @@
+import { AppInfo, AppInfoLocalization } from '@expo/apple-utils';
+import assert from 'assert';
+import chalk from 'chalk';
+
+import Log from '../../../log';
+import { logAsync } from '../../utils/log';
+import { retryIfNullAsync } from '../../utils/retry';
+import { AppleTask, TaskDownloadOptions, TaskPrepareOptions, TaskUploadOptions } from '../task';
+
+export type AppInfoData = {
+  /** The current app info that should be edited */
+  info: AppInfo;
+  /** All info locales that are enabled */
+  infoLocales: AppInfoLocalization[];
+};
+
+export class AppInfoTask extends AppleTask {
+  name = (): string => 'app information';
+
+  async prepareAsync({ context }: TaskPrepareOptions): Promise<void> {
+    const info = await retryIfNullAsync(() => context.app.getEditAppInfoAsync());
+    assert(info, 'Could not resolve the editable app info to update');
+
+    context.info = info;
+    context.infoLocales = await info.getLocalizationsAsync();
+  }
+
+  async downloadAsync({ config, context }: TaskDownloadOptions): Promise<void> {
+    assert(context.info, `App info not initialized, can't download info`);
+
+    // TODO: see why this type mismatch occurs
+    // @ts-ignore
```

Comment:
```suggestion
    // @ts-expect-error
```

so you're reminded to remove this